src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```
@@ -174,6 +174,21 @@ Copyright (c) .NET Foundation. All rights reserved.
       />
   </Target>
 
+  <!--
+    Trigger a warning if targeting a lower version of .NET Standard than is recommended
+  -->
+  <Target Name="_CheckForNotRecommendedNETStandardVersion" BeforeTargets="_CheckForInvalidConfigurationAndPlatform;Restore;CollectPackageReferences"
+          Condition="'$(_TargetFrameworkVersionWithoutV)' &lt; '2.0' and '$(TargetFrameworkIdentifier)' == '.NETStandard'" >
```

We should prefer the `VersionLessThanOrEquals` intrinsic function here.

```suggestion
          Condition="'$([MSBuild]::VersionLessThanOrEquals($(TargetFrameworkVersion), '2.0')) and '$(TargetFrameworkIdentifier)' == '.NETStandard'" >
```

---

LessThanOrEquals 1.6 or LessThan 2.0.